scripts/example_script.py

```diff
@@ -0,0 +1,241 @@
+import girder_client
+from girder_client import HttpError
+import json
+
+
+def print_http_error(err):
+    print ('Request Error:\nstatus:%s\nresponse%s\nurl:%s'
+           '\nmethod:%s ' % (err.status, err.responseText, err.url, err.method))
+
+
+def recurse_get_resource(client, parent_folder, resource_type,
+                         resource_list, parent_is_collection=False):
+    # now get all folders
+    folders = None
+    folder_id_list = None
+    if parent_is_collection:
+        try:
+            folders = client.listFolder(parent_folder,
+                                        parentFolderType='collection')
+        except HttpError as err:
+            print_http_error(err)
+            exit()
+        folder_id_list = get_field(folders, '_id')
+        if resource_type is 'folder':
+            resource_list.extend(folders)
+
+    else:
+
+        try:
+            folders = client.listFolder(parent_folder)
```

Comment:
This duplicated code between here and the other branch of `if parent_is_collection` can probably be combined, especially if the `parent_is_collection` parameter is changed.